### ISSUE TITLE
22.拯救

### ISSUE THREAD

**Issue by Pathwork-In-China** [Pathwork-In-China/Pathwork-Lectures#24]
22. Salvation
### 拯救
Pathwork Guide Lecture No. 22  |  January 31, 1958
心路指导讲座 第22篇 | 1958.1.31

22.1  Greetings in the Name of the Lord. I bring you blessings, my friends. Blessed is this hour.
以主之名问候你们。我为你们带来祝福，我的朋友们。这个时刻是受祝福的。

22.2  Angels of God always have had the opportunity to speak and manifest to human beings. Many people accept the possibility of manifestations by undeveloped spirits but deny that communication with more highly developed beings—whatever you may call them—should be possible. This is very shortsighted and illogical. The laws of nature and the universe must work alike for good or for evil, or for the many stages in-between. It is only a question of what conditions are prepared and fulfilled. If a person denies the possibility of any communication with any excarnated entity, this belief may be wrong, but at least it is consistent. However, to admit the possibility of one and exclude the possibility of the other is unreasonable.
来自上帝的天使一直都有机会同人类交谈并显现自己。许多人能接受“发展程度不高的魂灵们显现自己”的可能性，但拒绝接受“与发展程度更高的魂灵们——不管你们怎么称呼它们——沟通是有可能的”这种说法。这是十分目光短浅而不符合逻辑的。自然和宇宙法则必定对善与恶同样适用，对这两者之间的许许多多程度层次也应如此。问题只在于对相应条件的准备和满足。如果一个人否定与任何非降世状态实体沟通的可能性，那么尽管这种观点是错误的，但它起码是一致的。然而，接受一种可能性而排除另一种可能性的做法显然是不合理的。

22.3  There are many ways, indeed, to test with what spirits you are communicating. Anyone who wants to judge must, under all circumstances, take the trouble to study the vast subject of how to test spirits. Only then will that person be in a position to determine from where they come. I have spoken about this subject and will do so again in the future. For the time being, I only wish to say this:  If you still lack the necessary knowledge to determine with what spirits you are communicating and what laws are involved, you can ask your heart right now. You will get the answer to some extent, provided you do not let your doubts shade your feelings. If your feelings are receptive and open, you will be able to find and feel harmony, as well as love, patience, wisdom, and humility—or the opposite of all this. But your heart will only be able to speak to you and confirm what you wish to know if you are receptive, if you are open and put all prejudice aside for the moment.
确实，有很多方法可以测试与你交流的是何种魂灵。任何想要评估（魂灵们）的人，在任何条件下，都必须不怕麻烦地去研究“如何测验魂灵们”这个广阔的话题。只有这样，这个人才能处在一个有能力判定它们来自哪里的位置上。我之前讨论过这个话题，并且以后还会这样做。就目前来说，我只想说这些：如果你缺乏必要的知识来判定与你沟通的魂灵们是什么性质的，以及其中涉及到什么法则，你现在就可以对此扪心自问一下。你会在一定程度上得到答案，前提是你不让自己的疑虑影响了感觉和判断力。如果你的感觉是接受、包容而敞开的，那么你就有能力找到并感觉到和谐、爱、耐心、智慧和谦逊——或者是这一切的对立面。但只有当你处于一种希望接收的状态下，当你保持开放态度，并把一切偏见都先放在一边的时候，你的内心才能跟你交谈，并且证实你想知道的东西。

22.4  And now, my friends, I will continue the series of lectures I started. Last time I spoke about the creation of the earth, how it came into existence, and how humankind gradually evolved. I told you that the earth is a product or a picture, so to speak, of the longing of the spirits who have fallen and who have lived in darkness for a long time. I also said that the earth is equally a product of the longing of the spirits who remained in the world of God and who wanted to help their fallen brothers and sisters to come back into the divine light. Thus here on earth you can find beauty, love and harmony, as well as the opposite, with all the shades in-between. This is the proof that the earth sphere is a product of God’s world and the longing of the fallen spirits to reunite with God.
现在，我的朋友们，我会继续我之前开始的系列讲座。上次我谈到了地球的创造过程，它是怎样形成的，以及人类是怎样逐渐进化的。我告诉你们说地球是一个，从某种意义上说，由那些堕落的、并在黑暗中生活了很长时间的魂灵们（模糊）的渴望而产生的一个产物，或者说图景。我也说过，地球也同样是保留在上帝世界的魂灵们对于想帮助自己堕落的兄弟姐妹重新回到神圣光明中的渴望的产物。因此，在地球上你能发现美，爱与和谐，同时也能找到它们的反面，以及所有中间的色彩。这就是“地球位面是上帝世界和堕落魂灵对重回上帝的渴望所共同产生的产物”的证据。

22.5  On the earth sphere you have the possibility with your free will to develop and to decide which side to follow. In your own nature, you can of course find both currents:  the good current that once came from God in perfection and the evil current that was accumulated during and after the fall. Between these two currents the conscious self stands, able to decide either to take the line of least resistance, which is always man’s lower nature, or to follow his higher self, which is the difficult and narrow path.
在地球位面上，出于自己的自由意志，你们拥有做出“追随哪一边”这个决定的可能性。在自己的本性中，你们当然可以发现两种涌流都存在：曾经以完美的状态出自于上帝的善良涌流，以及在堕落发生的过程之中和之后积累起来的邪恶涌流。你的清醒自我站立在这两种涌流之间，它有能力决定要么追寻最小抵抗的道路（这总是人类的低级本性），要么去追寻他的高级自我（这是那条艰难而狭窄的道路）。

22.6  I have also mentioned that communication with the world of spirit has always existed, from the beginning of time when the earth sphere was relatively new. Then it was impossible for humans to communicate with God’s world because there were still too many unpurified currents and attitudes in the soul, too many evil desires and too much blindness to fulfill the necessary conditions. That is why God has always sent beings to the earth who were not fallen spirits. Some of the great prophets and so-called saints belonged in this category. Not only did they bring wisdom, love and purity with them to leave their mark upon this earth sphere, but because of their nature they could communicate with angels of God. This is briefly what I said in my last lecture to prepare you for what I have to say tonight.
我也提到了与精神世界的沟通从来都是存在的，从时间刚开始的时候、当地球位面仍处于相对新奇的状态时就是如此。那时候对人类来说与上帝世界的沟通是不可能的，因为当时（人的）灵魂中仍有太多不纯净的涌流和态度，有太多邪恶的欲望以及太多盲目性，以至于没法满足建立沟通联络的必要条件。这就是为什么上帝总是派一些没有堕落的存有们来到地球的原因。一部分伟大的先知和所谓的圣人都属于这个类型。他们不仅带来了智慧、爱和纯净性，从而在这个地球位面上留下了记号，而且由于他们的本性，他们可以与上帝的天使进行沟通。这就是我在上次讲座中简略谈到的内容，以此为我今天要说的内容做准备。

22.7  I should like to tell you what salvation by Jesus Christ really means. Very few people indeed realize its full significance, least of all the organized churches who have almost completely misunderstood what salvation is. Many people believe that Christ died on the cross for the sins of everyone else and as a result no one is responsible or accountable for their sins, faults and weaknesses, for Christ has atoned for them through His death. This, of course, my friends, cannot be so. It would be utterly senseless. After the explanation of the real story of salvation you will not only see that this is a comfortable misunderstanding, but you will also clearly perceive how this misunderstanding could have come about.
我想要告诉你们“通过耶稣基督得到拯救”的真正含义。确实只有很少的人意识到了其完整的重要性，更别说所有那些有组织的教会了，它们几乎完全误解了拯救是什么。许多人相信基督是因为所有其他人所犯的罪而死在了十字架上，因此没有人需要为自己的罪、过错和弱点负责了，因为基督通过祂的死而为所有人赎罪了。我的朋友们，这肯定是不对的。（如果真的是这样的话，）那将会是毫无意义的。在听到拯救的真实故事后，你不仅会看到这是一个合理的误解，而且还会清晰地觉察到这种误解是如何形成的。

22.8  I have also mentioned that salvation was not only accomplished on this earth sphere but in every sphere of existence. Long before the earth sphere existed and after the so-called Fall of the Angels, which I described some time ago in greater detail, God’s plan was ultimately that every single one of those fallen beings should have the means to come back to Him, back to the light and harmony. But it was essential that God’s laws never be broken, not even for the purpose of bringing back the fallen creatures. This was, indeed, a very difficult task to follow through.
我之前也提到过，拯救不只是在地球位面上得到完成，而是所有存在位面上完成的。在地球位面形成前很久的时候，在所谓的天使堕落（对此我之前已经较为详细地描述过）发生之后，上帝最终的计划是那些堕落的存有们每一个都应该有回到祂、回到光明与和谐中的手段。但同样关键的是上帝的法则不能被打破，哪怕是以把这些堕落的生物带回来的目的也不行。这，确实是一个很难实行的计划。

22.9  I also explained that each being put into creation by God was created perfectly in one way. Each being represented one divine aspect. The purpose was for each spirit to extend this perfection into other realms. Let us say one being was perfect in love, another in wisdom, and so forth. The aim was to use the divine power that we all once had to make ourselves perfect in every other respect and thus eventually become godlike. By so doing, additional worlds of beauty would come into existence—spiritual worlds—for as you know, every thought, every feeling, every ambition, and every act is formed in spirit and thus creates a world.
我也解释了，每个被上帝投入宇宙造物中的存有都在某一方面是完美的。每个存有代表一个神圣侧面。其中的目的在于让每个魂灵把这份完美延伸到其他领域。让我们假定，一个存有拥有完美的爱，另一个拥有完美的智慧，如此等等。目标在于，利用这份我们都曾经拥有的神圣力量把自己变得在每个其他方面都完美的状态，从而最终变得类似上帝。在这个过程中，会有额外的美丽世界——精神世界——形成，因为如你们所知，每个想法，每个感觉，每个野心，每个行为都在精神中形成，因此会创造出相应的世界。

22.10  You also know that a number of spirits have used their divine power for this purpose, and a number of other spirits have used theirs in the opposite way. This was the origin of the fall. After the fall, Christ, who of course existed in the spirit world long before He was born as man, organized all the spirits in God’s world to use all their strength and perfection in their particular fields to help the Plan of Salvation. In other words, the pure spirits, instead of continuing and extending their own growing perfection, postponed this ultimate goal for the purpose of using their powers to arrange and to work for the Plan of Salvation. And this plan exists on every plane. Now, of course, I am still talking of the divine spheres.
你们也知道有一定数量的魂灵们已经把自己的神圣力量用于这个目的，而也有一定数量的其他魂灵们把它们的力量以相反方式使用。这就是堕落的起源。在堕落发生之后，基督（在祂作为一个人出生之前早就存在于精神世界中）把所有上帝世界中的魂灵们组织起来，使用它们的各自领域的力量和完美属性来帮助拯救计划。换句话说，那些纯净的魂灵们，停止了继续延伸它们自己不断生长的完美的过程，推迟了这个最终目标，目的是使用它们的不同力量来安排并为救赎计划而工作。这个计划存在于所有位面上。当然，我这里指的仍然是神圣位面。

22.11  In the spheres of darkness, some time had to go by before anything could be done. Enough spirits had to feel a longing for light before lighter worlds, still in the realm of Lucifer, came into existence. Without this longing, unconscious and blind as it was at the beginning, nothing could change, regardless of what would have been arranged by the world of God.
在那些黑暗位面中，必须在一段时间经过之后才有发生任何变化的可能。要有足够多的魂灵们感到对光明的渴望，才能使一些相对光亮些的世界逐渐开始在路西法的领域中形成。假如没有这种渴望（尽管一开始它是潜意识又盲目的），就不会有任何改变，不管上帝的世界做出怎样安排都没有用。

22.12  In your language, millions and millions of years went by before, due to this longing, the earth sphere gradually came into existence. More souls came to live on earth because they were ready to do so, low as their development still was. More general and individual development then proceeded. Merely by living on the earth sphere, they came, for the first time after the fall, into contact with something divine, subdued as this manifestation may have been for the most part.
用你们的语言表述，数百万年过去了，由于这种渴望，地球位面逐渐形成。更多的灵魂来生活在地球上，因为它们准备好这样做了，尽管它们的发展阶段仍然很低。这时候发生了更加普遍、更加个性化的发展。仅仅通过生活在地球位面上，它们，在堕落发生后第一次，与某些神圣的东西建立了联系，尽管这种东西之前很大程度上是被压制的。

22.13  In the meantime, Christ was busy preparing and working in the spirit world of God, planning ahead and sending various pure spirits to live on earth. He also organized teachings for the pure spirits now incarnated to bring to mankind, either through inspiration and guidance, or through communication with God’s world. It is impossible for you to imagine how minutely everything had to be worked out, how painstaking this work was so that everything would accord with the divine laws of justice.
与此同时，基督忙于在上帝的精神世界中做准备工作，提前计划着派出许多不同的纯净的魂灵们去地球上生活。他也为现在正在降世的纯净魂灵们组织了将要教授给人类的教导，要么通过灵感和指引，要么通过与上帝世界的沟通。对你们来说要想象出一切是怎样分毫不差地运行，以及这份工作是多么的煞费苦心才能使一切都符合神圣法则的正义是不可能的。

22.14  At that time, no matter how far human beings developed spiritually, when returning to the beyond, they were still under the dominion of Lucifer. For, as I explained last time, every divine aspect was turned into its opposite quality. Therefore, free will, which is divine, was turned into domination. And, of course, Lucifer would not give up the dominion he held over his followers. If, for instance, a human being, due to a changing attitude and growing harmony with God, would begin to produce light and beautiful spheres in the spirit world, even these spheres still belonged to the kingdom of Lucifer because he did not relinquish his power over this person. Furthermore, no one was at that time far enough developed to produce only spheres of light. People would produce and own several spheres, harmonious and disharmonious ones.
在那个时候，不管人类在精神方面取得多少进展，当回到之上的时候，他们仍然处于路西法的统治之下。因为，如我上次解释的，每个神圣位面都转到了它的对立品质。因此，自由意志，作为神圣品质，被转入了统治支配。而必然的是，路西法不愿意放弃他对他的追随者们拥有的统治权。比如说，如果一个人由于态度的转变和与上帝的关系变得和谐，而开始在精神世界里产生光明和美丽的位面，但哪怕是这些位面也仍然属于路西法的王国，因为路西法并没有放弃自己对这个人的掌控。而且，那时候没有人能取得足够的进展从而只产生光明的位面。人们会产生自己的数个位面，其中有和谐的也有不和谐的。

22.15  This happens, incidentally, with each one of you and with every human being. Wherever there are faults, weaknesses and blindness, corresponding spheres come into existence. Wherever you are pure and purified, you create beautiful spheres. And you will not only own the best, but also the worst of that which you have built. Any relatively highly developed human being might thus inhabit a number of light spheres, but even these spheres were still under the dominion of Lucifer as long as the salvation work in this respect was not finished. By the way, what you call hell is not only one sphere of dire darkness and misery. As there are many gradations in the divine spheres, so there are similar gradations in the Luciferic ones.
偶尔的，这种情况会发生在你们每个人以及所有人身上。只要（人内心）还存在过失、弱点和盲点，就会形成相应的位面。每当你进入纯净、净化的状态，你就创造出美丽的位面。因此你不仅会拥有最好的那些，也会拥有自己创造的最坏的那些。任何发展程度相对高级的人可能因此居住在一系列的光明位面中，但只要拯救工作在这个侧面还没有完成，那么哪怕这些位面在当时也仍然处于路西法的掌控之下。另外，你们称为地狱的东西不是单单一个充满极端黑暗和痛苦的位面。就像神圣位面中有许多等级阶段一样，在路西法派系的位面中也有相似的等级阶段划分。

22.16  When enough beings were ready and cognizant of God, and consciously desired to have complete union with God, the time was ripe for the most important part of the Plan of Salvation to take place, which Christ took upon Himself. He had a reason beyond His infinite love and compassion with all His fallen brothers and sisters.
当有足够多的存有们准备好并觉察到上帝，同时清醒地渴望与上帝完整联合时，开始进行拯救计划中最重要的一部分的时机就成熟了，而基督亲自承担起了这项工作。祂对帮助自己所有那些堕落的兄弟姐妹们有着超越了祂的无限爱和同情的理由。

22.17  During the process of the fall, the first spirit who fell, Lucifer, developed an intense jealousy of Christ. Thus it was logical that Christ Himself should prove His love by His great sacrifice and work, not only to all the other fallen creatures, but also to Lucifer himself, who through Christ’s deed alone would one day in the far future find it possible to return to God and thus find ultimate happiness. God made Christ the King of the Universe and as such Christ possessed not only the highest privileges, but also the strongest responsibilities. By carrying the heaviest burden alongside His exalted position, He gave another example for the world to follow.
在堕落的过程中，第一个堕落的魂灵，路西法，培养出了一种对基督的强烈嫉妒。因此符合逻辑的做法是，基督应该亲自通过自己的伟大牺牲和工作向不仅是所有的堕落生物们，而且也向路西法他自己证明祂的爱，而路西法仅仅因为基督的这种行为，将有朝一日可以在遥远未来的某一天发现回归上帝变得可能，并因此找到最终的幸福。上帝使基督成为宇宙之王，而作为宇宙之王，基督不仅享有最高的特权，也承担了最重的责任。通过同时承担起最沉重的负担和祂高贵地位做法，祂给了世界又一个可以遵循的榜样。

22.18  Thus, when the time was ripe, Christ faced Lucifer. Now, my friends, I must ask you not to think that all this could not have been that way because it sounds all too human. All you have and know as human beings, not only regarding subjects and objects in abstract and concrete ideas and in language, but also in any sort of forms you know is only a limited imitation of what existed in spirit before this material world and in much greater variety.
就这样，当时机成熟时，基督面对了路西法。现在，我的朋友们，我必须请求你们不要认为“所有这些不可能是以那种方式发生的，因为它听起来太像人类了”。你们头脑中的一切关于人类的概念，不仅仅是那些与主题和物品的抽象的、具体的概念和语言中的概念，也包括任何你们所知的形式，这些都只不过是对“在这个物质世界形成之前以丰富得多的形式存在于精神世界中的东西”的很局限的模仿而已。

22.19  Human beings think, when we mention that spirits talk or possess certain objects, that this is too human and too concrete. Yet, in spirit, as I have repeatedly said, everything is concrete, everything is form. In your world only material objects have form and the so-called abstract things do not have form, for they are invisible to you. This is not so in spirit. Love is a form. When you have a beautiful thought it will create one form. When you have a wicked thought, it will create another form, a concrete form for us.
人们认为，当我们提到魂灵们会互相交谈或者拥有某些物品，这听起来太像人了、太过实体化了。然而，在精神世界中，如我反复所说，一切都是实际、实体的，一切都有形。在你们的世界里，只有物质物体才有形状，而所谓的抽象的东西没有形状，因为它们是看不到的。在精神世界中情况不是这样。爱是一种形态。当你有一种美丽的想法时它会创造一种形态。当你产生一个邪恶的想法是，它会创造另一个形态，对我们来说这是具体有形的。

22.20  I beg you to keep this in mind and not think what I am telling you is childish because Lucifer and Christ would not talk together like two human beings. It may not be in exactly the same way as when two human beings talk. The procedure may be different. It is a spiritual procedure. This, of course, is impossible to translate into human language. Therefore, the language I have to use must be limited to your understanding.
我请求你们（在做出判定前）考虑到这个情况，而不要因为（听到）路西法和基督会像两个人一样交谈而觉得我告诉你们的东西很幼稚。对话的过程也许不完全像两个人类个体谈话那样进行。其中的过程是不同的。这是一个精神/心灵的过程。当然，这个过程是无法翻译成人类语言的。因此，我使用的描述性语言必须因为你们的理解力而受到限制。

22.21  So, to continue:  Christ would face Lucifer and tell him:  “Now, there are so and so many spirits who do not wish to remain faithful to you. They desire to go back to God. Therefore, you should set them free.”  Lucifer would not agree to that. He maintained that he did not recognize divine law and would use his power as he saw fit.
那么，继续刚才的过程：基督会面对路西法并告诉他：“现在，有如此多的魂灵们不想对你保持忠诚。它们渴望回归上帝。因此，你应该给它们这样做的自由。”路西法对此不同意。他维持了之前的态度，也就是说他不认可神圣法则，并且会根据自己的意愿使用自己的力量。

22.22  So Christ said:  “In this case there must be a war between us, between your forces and the forces of the divine world.”  The chances would be evenly distributed, which means that the divine forces must be numerically in the minority for the simple reason that the forces of good are infinitely stronger than the forces of evil, perhaps twenty to one. If you have one absolutely purified being against twenty very impure beings, the strength of this one pure being would outweigh the strength of the twenty impure ones.
于是基督说；“既然这样我们之间必须有一场战争，战争的双方分别是你的力量和神圣世界的力量。”双方获胜的概率应该均等，也就是说神圣力量必须是数量上的少数，原因很简单，善良的力量比邪恶力量强无数倍，（从数字上来说）也许是二十比一。如果你有一个绝对纯净的存有对抗二十个非常不纯的存有们，这一个纯净存有的力量会压过二十个不纯净存有们的力量。

22.23  Lucifer said:  “Even if such a war took place and even if the divine forces won and took my power away, I would still not recognize the law of God as being just.”  As you know from my previous lectures, this issue constituted an essential part of the Plan of Salvation, since no one should be eternally damned, not even Lucifer himself. And so that no eternal damnation would ever be possible, Lucifer himself would have to admit at all times the absolute justice of the divine laws.
路西法说：“就算这样一个战争发生了，就算神圣力量赢了并且夺走了我的力量，我仍然不会认可上帝的法则是正义的。”如你们据我之前的讲座所知，这个问题构成了拯救计划中的关键部分，因为没有谁应该受到永恒的谴责，哪怕路西法他自己也不应该被这样对待。因此永恒的天谴永远是不可能的，路西法自己也必须在任何时候都承认神圣法则的绝对正义。

22.24  Therefore, Christ asked him:  “In what way would you consider the divine powers to be just?”
因此，基督问他：“在什么情况下你会认为神圣力量是公正的？”

22.25  And Lucifer answered:  “I would fight such a war if one being—from the world of God, if you wish—would live on earth like a man, without any protection or guidance from the world of God at crucial times, with a great part of his knowledge dimmed out, with matter standing in the way, and would yet remain faithful to God in spite of my temptations and in spite of the most difficult conditions possible. I would offer this person every worldly power and release from all hardship if he forsook God. If he remained faithful to God under such conditions—which I doubt very much, in fact, I say it is impossible—then I will have my battle with you and I will recognize God’s laws as being utterly just.”
路西法回答说：“我会接受这样一场战争，前提是有一个存有——来自上帝的世界，如果你愿意的话——要像一个普通人一样生活在地球上，在关键时刻不能有任何来自上帝世界的保护和指引，而且他知识的相当大部分要被变得模糊暗淡，受到物质的阻拦，同时还要在我的诱惑以及最最困难的条件下保持对上帝的忠实。我会给这个人一切世俗的力量并撤销全部的艰苦遭遇，前提是只要他抛弃上帝。如果他在这种条件下仍然对上帝保持忠实——我对此严重怀疑，事实上，我说这是不可能的——那么我就会接受与你的战斗，而且我会同意上帝的法则是彻底公平的。”

22.26  You must know, my friends, that every living being has at all times guardian spirits from the world of God. But some people’s attitude may keep these spirits from coming too close. Nevertheless, they are there, even if only in the background, watching that nothing befalls their protege that is not according to God’s laws of justice or that the person may be too weak to endure. To be left alone without the support of God’s spirit world on this earth sphere, and in addition having to resist all attacks, challenges, hardships, and temptations that the powers of darkness could think of seemed indeed an impossible task to fulfill. No human being had ever had to go through anything remotely like it. Therefore, Christ cannot be compared with any other person who has ever lived, no matter how pure or how wonderful the teachings may have been. Christ has shown in deed and in fact what others have taught, and He did it under infinitely more difficult circumstances than anyone else ever had to bear.
你们必须知道，我的朋友们，每个生活着的存有在任何时候都有守护魂灵，它们来自上帝的世界。但有些人的态度可能会阻止这些魂灵们靠近自己。但不管怎么说，它们都在那里，哪怕仅仅处在背景中，确保没有任何不符合上帝的公正法则的的事情，或者对那个人来说没法承受的事情会降临在自己的被保护者身上。要在没有上帝世界的支持下孤单地留在这个地球位面上，同时必须克服各种攻击、挑战、艰苦条件以及黑暗世界能够想出来的各种诱惑，这似乎确实是一个不可能完成的任务。还没有哪个人类个体曾经必须经历任何与此类似的情况。因此，基督不能与任何其他生活过的人同理而论，不管之前的教导是多么地纯粹或奇妙。基督用行动和事实表明了别人教导的东西，而且祂是在比任何其他人必须承受的条件恶劣万倍的情况下做到这一点的。

22.27  So these were the conditions Lucifer set up for him to recognize God’s laws as being just. If this seemingly impossible task were really to be fulfilled, then the battle could take place. Should Lucifer lose the battle, then Christ could make His terms and Lucifer would not doubt God’s justice in any respect. This then was the plan. And Christ took it upon Himself for the above-mentioned reasons, though Lucifer did not specify that it had to be Him.
所以这就是路西法为自己设定的要把上帝的法则认可为公正的条件。如果这个似乎不可能的任务真的被完成了，那么战斗就可以进行。如果路西法输掉了战斗，那么基督就可以宣布祂的规则，而路西法将无法在任何方面怀疑上帝的公正。于是计划就是这样了。而基督出于之前提到的原因亲自担负起了这个任务，尽管路西法没有明确指定一定要是祂来完成。

22.28  My friends, if you study all the Scriptures from this point of view, you will get an entirely different understanding of them. I am quite sure that the reason for the life and death of Christ will now make sense to you. There would not be any sense in Christ dying on the cross for sins others have committed. If you have committed a sin, you yourself have to straighten it out and no one else can or should do it for you. If someone else were to do it for you, you would not gain purification. You would not receive the strength through the process of self-purification which alone will protect you from committing sins again. As long as the evil root is not torn out, it must again produce impure fruit. Only you can tear out the roots of your evil. Therefore, that was not the reason Christ suffered and died.
我的朋友们，如果你们以这种角度研究所有的圣典，就会从中得到一种完全不同的理解。我很确信，你们现在应该对基督的生活和死亡的原因很清楚了。基督因为别人犯下的罪而死在十字架上是说不通的的。如果你犯了罪，是你自己必须把它理顺，而没有任何其他人可以或应该为你做这件事。如果某个别的人替你做了这件事，你不会（从中）获得净化。你不会从自我净化的过程中获得力量，而只有这份力量才能使你不再犯罪。只要邪恶的根源没有被铲除掉，它一定会再次产出不纯净的果实。只有你能铲除自己内心的邪恶根源。因此，那不是基督受难并死去的原因。

22.29  You will also understand why Christ was left completely alone for a long time. Naturally, as a man, He did not have the same knowledge He had as a spirit. If He had had that same knowledge, the task would not have been so difficult. He did of course possess some knowledge since He is the highest being in creation. In addition, He had a great deal of spiritual strength and wisdom. However, there would be no purpose to life on earth at all if—and this applies to everyone—the same spiritual knowledge were available in the flesh as when one is not incarnated.
你们也会理解到为什么基督在一段时间内被彻底孤立。很自然的是，作为一个人，祂当时没有自己作为一个魂灵时所拥有的的知识。如果祂当时拥有同样的知识，这个任务就不会那么困难了。不过祂当然拥有一些知识，因为祂是宇宙造物中最高的存有。除此之外，祂还有相当的精神力量和智慧。然而，如果在肉身时拥有的知识与当一个人处于非降世状态时拥有的知识一样多的话，在地球上生活就没有任何意义了（这一点对所有人都适用）。

22.30  So Christ did not know exactly what was involved while He lived on earth. In the course of the years, He received some knowledge, and He had a vague idea—just as any one of you might have a vague idea—of the task He was to fulfill. What may come of it, how it will end, what the exact meaning is, you will not know—and He did not know that either. He was not supposed to know it while incarnated. After a certain time all the angels of God had to leave Him. They were with Him for some time of His life, but were not present when the really difficult task began.
于是基督当时不完全知道自己生活在地球上时，到底涉及到怎样的情况。在数年的过程中，祂接收了一些知识，而且祂有一个模糊的概念——就像你们任何人可能有的模糊概念一样——有关自己应该完成的任务。事情会怎样发生，怎样结束，到底有什么具体含义，你们不会知道——而祂也不知道。祂在降世状态中就是不应该知道的。在某个时间点以后，所有上帝的天使都必须离开祂。它们在祂生活中的一段时间内是陪伴在周围的，但当真正困难的任务开始时它们是不在的。

22.31  As I have explained to you, the teachings He brought were important and wonderful, but this was an additional facet of His life. It was a side benefit. Whenever anything happens in strict accordance with the will of God, not just one good reason and purpose exists, for many factors play a role and many good purposes are fulfilled with one divine deed. This again applies to everyone. However, simply to bring the teachings was not His full reason for living as a man. Beautiful as the teachings are, they were not new. In essence, other people had also brought the same teachings. He adjusted them according to His times and in consideration of the ever-evolving development of humankind, but that was all.
如我已经解释给你们的，祂带来的教导是重要又美妙的，但这只是祂生活的一个额外侧面。它是一个附带效益。每当任何事严格遵循上帝的意志发生时，不仅只有一个好的原因和目的存在，因为许多因素都在起作用，因此，一件神圣行为会完成数个好的目的。这个情况同样对每个人都适用。但是，简单地带来那些教导并不是祂作为人进行生活的完整原因。尽管那些教导很优美，它们并不是新鲜的。本质上来说，之前有其他人也带来了相同的教导。祂把这些教导根据当时的世代以及从来都在进化的人类发展现实而做出了调整，但这不是事情的全部。

22.32  The task was, as I explained, that He—left quite alone and cut off completely from the world of God—had to resist the temptations of Lucifer, who put the greatest effort imaginable into his goal to cause Christ to fall. He used every device possible, and in so doing he organized all his helpers. Believe me, Lucifer is by no means stupid, even though he certainly lacks wisdom and insight. Nor is he without great resources in his own dark powers.
要完成的任务是，如我解释的，祂——在孤立无援、与上帝的世界彻底隔绝的情况下——必须抵抗路西法的诱惑，而路西法为了让基督堕落，花费了可以想象的最大努力。他利用了一切可能策略，组织起了他全部的帮手。相信我，尽管他肯定缺少智慧和洞察力，路西法绝对不愚蠢。同时他在自己的黑暗力量方面也是相当足智多谋的。

22.33  On the one side Christ experienced nothing but suffering—both physical and psychological, as well as humiliation, the extent of which you cannot image. The humiliation and psychological suffering were a great deal worse than the physical suffering, as bad as that was. On the other side, He was offered all the temptations of the world of darkness.
一方面，基督除了煎熬外没有体验到任何其他感觉——这种煎熬是身体和心理上双重的，除此之外还有羞辱，这种羞辱的程度是你们无法想象的。这种羞辱和心理上的煎熬要比身体上的煎熬糟糕很多，尽管身体上的煎熬本身已经糟糕了。另一方面，祂收到了黑暗世界的各种诱惑。

22.34  Of course, Christ was what you would call psychic to the maximum degree. His mediumistic qualities were so strongly developed not just in one respect, but in every respect, that they were greater than anyone else’s before or after. This was an advantage as long as God’s world was close to Him, but when He was cut off from it, this was merely an additional hardship, for all the manifestations coming to Him originated from the world of darkness.
当然，基督是一个会被你们称为拥有最大限度通灵能力的人。祂的传导品质发展的非常强力，不是在一个方面，而是在每个方面，以至于这些能力比任何之前或之后来的人的能力都强。这种特征在上帝的世界与祂靠近的时候是一种优势，但当祂被与之隔绝时，这就变成了一个单纯的额外艰难，因为所有到祂那里的信息和显现都是来自黑暗世界的。

22.35  Clairvoyantly, He came in contact first with high emissaries from the Luciferic world and later with Lucifer himself, who made himself appear as a beautiful being, offering Christ all the worldly advantages He might desire and instant release from all His sufferings, if He accepted Lucifer and gave up His idea of God. Lucifer taunted Him in the worst moments of His sufferings:  “Where is your God of love and justice?  If He existed, would He allow His beloved son to go through all of this?  If your God cannot offer you more, are you not better off with me?  Look what I have to offer you. Your God can only offer you intense suffering and hardship in every possible respect.
在额外视觉状态下，祂一开始遭遇了来自路西法世界的高级使者，后来又遇到了路西法本人，他以一个美丽的存有的形象出现，向基督提出祂可能想要的一切世俗的好处并且立刻可以撤销祂的全部苦难，只要祂接受路西法并放弃祂的上帝的概念。路西法在祂煎熬中最痛苦的时刻嘲笑祂：“你充满爱和正义的上帝在哪呢？如果祂存在的话，祂会允许自己挚爱的儿子经历这一切吗？如果你的上帝不能给你提供更多，那你跟从我不是更好吗？看看我能给你提供的东西。你的上帝只能给你带来每个可能方面的强烈痛苦和艰难。”

22.36  Can you imagine what this meant?  If Jesus had known the exact significance of His task, it would not have been half as difficult to resist. But this was precisely the point. To have doubts at these crucial times, doubts about everything, about His true identity, and about there being any wise and good purpose in undergoing all the hardships which He could not understand at the time—in short, about everything He had learned in the previous years—was inevitable. Often He wondered whether He was not under some illusion and whether all His previous knowledge was not the product of imagination. During these times of doubt Lucifer would instantly be at His side and strengthen such thoughts.
你能想象这意味着什么吗？如果耶稣知道祂的任务所包含的确切的重要性，想要抵抗这一切就根本没有那么困难了。但这正是事情的关键。在这些关键时间点上产生疑虑，对于一切的疑虑，包括祂的真实身份和经历当时祂没法理解的这一切艰苦情况背后有没有任何明智的、有益的目的——总之，关于祂在过去的岁月中学到的一切——这些疑虑的产生是不可避免的。经常地，祂会奇怪自己是不是遭受了某种幻觉的影响，或者祂之前学到的一切知识是不是疯想的产物。在这些充满疑虑的时刻路西法会立刻出现在祂旁边并增强这种想法。

22.37  It is easy to perceive how extremely hard it must have been for Him—being a man and having matter between Him and absolute truth—to remain faithful to God and not give in to these temptations aggravated by hardships. If the conditions of His task had not been such that even Christ might have doubted at times, His task would not have been so infinitely magnificent. Therefore, Christ had to have the same obstacles of matter as all other human beings, but his were intensified to a maximum degree. Material substance is a curtain and man has to grope to open that curtain. Jesus Christ had to do the same, but in conditions of which you can only vaguely appreciate the extreme difficulty, even with these explanations. To remain on the right path under these circumstances without fully understanding it—my friends, you cannot really know what it meant. And having the humility, in spite of all the passing thoughts of doubt, to put God above everything, even above His suffering, and above His not understanding why, was the task. It indeed seemed almost impossible that anyone could do it. But Jesus Christ did!
一个人可以很轻易地感受到当时的情况对祂是多么极端地不利——作为一个人，并且在祂和绝对真理之间隔着物质的阻碍——保持对上帝的忠实而不屈服于被这些艰难情况加剧过的诱惑。如果祂的任务所包含的条件不是这么严酷，以至于甚至基督自己在一些时刻都产生了怀疑，祂的任务就不会如此显赫了。因此，基督必须像所有人类一样承受物质造成的障碍，但祂的情况被增强到了最大的程度。物质材料是一道幕布，而人类必须在摸索中才能打开那道幕布。耶稣基督也必须这样做，但祂所面对的条件你们只能在这些解释下才能模糊地体会到其中的极端困难。要在不完全知情的情况下，在所有这些因素的影响下，保留在正确轨道上——我的朋友们，你们无法真正知道这意味着什么。而在所有经过的充满疑虑的想法中，仍然有足够的谦逊谨慎而把上帝置于一切之上，甚至置于祂的苦难之上，还要置于祂无法理解事情的原因这个现实之上，这就是祂的任务的本质。要任何人做到这一点在当时确实是显得不可能的。但耶稣基督做到了！

22.38  By doing this, Christ not only fulfilled the conditions by which the world of darkness could never at any time claim God’s laws to be unjust, but at the same time He set an example for everyone born after Him. So when you are in suffering and you do not understand why, think of Jesus Christ within the setting of the true story of salvation. Then bring yourself to imagine His sufferings as something real, not some imaginary legend, as real as your own sufferings, only much worse. Then it will be so much easier for you to follow in His footsteps and remain humble, letting God take over.
通过做到这一点，基督不仅完成了相应的条件，使黑暗世界在任何时候都无法声称上帝的法则是不公正的，而且同时祂还为之后出生的所有人都做出了一个榜样。所以当你在经受痛苦却又不理解为什么时，想一想耶稣基督在真正的拯救故事中的情况下的处境。然后让自己想象下祂的受难是一种真实的经历，而不是什么想象出来的传说，这种苦难就像你自己的苦难一样，只不过要严重很多。这时对你来说追随祂的脚步并保持谦逊态度，并让上帝接手就会容易多了。

22.39  Immediately after Christ had successfully completed His task on the earth sphere, a number of so-called “miracles” took place on earth, showing humankind that a major phase in the history of creation was over and an important new phase was to begin. I could talk on for hours and hours about His life on earth, my friends, about His sufferings and about His death. But if you read the Bible now, your own imagination can give you a better idea than before of the deep significance and the reality of all this.
当基督在地球位面上成功完成了祂的任务之后，立刻发生了一系列的所谓“奇迹”的事件，它们向人类表示出一个信号，那就是宇宙造物历史上一个重大的阶段结束了，另一个新的重要阶段将要开始了。关于祂在地球上的生活、关于祂的苦难以及关于祂的死亡我可以说上许多许多小时，我的朋友们。但如果你现在读《圣经》，你的想象力可以给你一个比之前更好的关于这一切事情的重要性和真实性的概念。

22.40  After Christ’s physical death, He returned to the world of spirit. Having fulfilled the conditions with a relatively small number of specialized spirits He fought a spiritual battle in the world of darkness.
在基督身体的死亡之后，祂回到了精神世界。在与一些数量相对较少的专家魂灵们满足了之前设定的条件后，祂在黑暗世界中进行了一场精神/心灵战斗。

22.41  That spirits should have wars, my friends, may sound again too human for you. Where do you think your wars come from?  They are only an outpicturing of spiritual war. Of course, a spiritual war does not take place in exactly the same manner as a material war on earth, but the essence is nevertheless there. How it happens is again impossible to describe because you lack the perception and understanding as I lack the full ability to express myself in human language. So I can only describe it in a somewhat condensed way that may sound symbolic and may be symbolic to a certain extent.
魂灵们居然会开展战争，我的朋友们，可能这在你们听起来太过于人类化了。你们认为自己的战争起源于哪里？它们只是精神/心灵战斗的延伸而已。当然了，一场精神/心灵战争并不完全以与发生在地球上的物质战争一样的形式发生，但（战争的）本质不管怎么说是在的。它是怎样发生的同样也是无法向你们描述清楚的，因为你们缺少相应的感知手段和理解力，而我缺乏以人类语言充分表达我自己的能力。所以我只能以某种浓缩的方式描述它，以至于这种描述听起来有象征意味，并且在一定程度上确实具有象征性。

22.42  So a war took place between Christ and Lucifer. You have to use your inner vision and imagine it as though a war with guns or spears, as on earth, had taken place in exactly that form. Of course, this was not the way it was. However, there was a spiritual war. Again Lucifer had to admit the justice of the ways of God’s world, for, as mentioned before, Christ came to fight under equal conditions. It would have been in His power not to take any risks by using greater strength and more helpers. However, He did not and this was for the same reason that He undertook life on earth for God’s justice to be preserved—even in the eyes of Lucifer. The chances were even and this was so apparent that not even Lucifer could deny it. That was important, for the plan was, and is, that Lucifer himself must ultimately come to the point when he too will return to God as the very last of all the fallen creatures, since he was the first to turn away from the laws of God.
于是一场战争在基督和路西法之间展开。你必须利用自己的内部视觉来把它想象成好像一场地球上以枪和矛为武器的战争，以完全相似的形式发生。当然，这不是实际发生时的样子。但是，当时确实是有一场精神/心灵战争。路西法再一次地必须承认上帝世界规则的公正性，因为，如前面提到的，基督以对等的条件来参与战争。祂有能力不冒任何风险，使用压倒性的力量并带来更多的帮手。但是祂没有这么做，理由与祂之前进入地球上生活一样，那就是为了保存上帝的公正性——哪怕在路西法眼中也是如此。双方取胜的概率是对等的，这是如此的明显以至于连路西法也没法否认。这一点很重要，因为当时的计划是，并且仍然是，要路西法自己也必须最终到达一种状态中，使他作为堕落生物中最后的一个回归上帝，因为他是第一个摒弃上帝的法则的。

22.43  Jesus Christ fulfilled the Plan of Salvation in every sphere. His task varied in each of the numerous spheres, because each was different: in the world of God where the manifold preparations were made, on the earth sphere, and in the world of darkness. After the battle was over, new conditions were set up. They have reigned ever since.
耶稣基督在每个位面上都完成了拯救计划。祂在数个位面中的每一个位面上都有不同的任务，因为每个位面都不一样：在上帝的世界中要做出纷繁复杂的准备，在地球位面，以及在黑暗世界。当战争结束后，新的条件被建立起来。这些条件沿用至今。

22.44  In your history you will read that on the third day, after His descent into Hell, Christ went up to Heaven. The various details that were preserved in Scripture confirm in a way all this for you, although the time element is not quite exact. Time is always a “translation,” so to speak, for in spirit, time—if there is such a thing—is individual, psychological, and very different. But this does not matter, for humankind has made a symbol of these three days.
在你们的历史中你会读到，在第三天，在祂沉入地狱之后，基督升入了天堂。圣典中保存下来的许多细节会以一定方式向你证实这一切，尽管其中的时间因素不是非常准确。时间（的概念）一直是一种（如果可以这么说的话）“翻译”，因为在精神/心灵中，时间——如果真有这样一种东西的话——是个体化的、心理的，并且是非常不同的。不过这不重要，因为人类文明把这三天变成了一种象征。

22.45  The new conditions meant that all human beings were given the opportunity to turn to God during their development on earth, going from one life to the other. Lucifer kept all the rights to tempt humans to succumb to him by succumbing to their own lower nature. Should they resist, they would no longer be subjects of the Luciferic world, for the doors were now open to unite with their Creator and inhabit the divine worlds once more. Even the traps and temptations that Lucifer could use were, from that time on, limited. In accordance with divine law, God’s spirit world now has the right to interfere. The divine laws must be observed in exactitude, the activities of the powers of darkness are limited and must stand ultimately under the jurisdiction of God.
新的条件意味着所有人类在处于地球上的发展过程中，以及从一次生命转入下一次生命时，都获得了转向上帝的机会。路西法保留了一切诱惑人类，并让他们通过屈服于自己的低级本性而屈服于他的权利。如果他们做出抵抗，他们将不再从属于路西法派系的世界，因为现在大门已经敞开，使他们有机会与自己的造物主重聚，并再一次栖居在神圣世界里。就连那些路西法可以使用的陷阱和诱惑手段，也从那个时候开始受到了限制。按照神圣法则的要求，上帝的精神世界现在有权进行干涉。神圣法则必须得到精准严格的遵守，黑暗力量的活动得到了限制，并且必须最终处在上帝的管辖权之下。

22.46  For Lucifer to still possess a certain amount of freedom is necessary not only for the now so often explained reason that he must always recognize divine justice, but also as a necessary means for development. Evil has to be tasted to the brim in many cases before it can be overcome by free will and the being’s own initiative. The desire to overcome must grow through ever-mounting enlightenment in each individual’s soul, and this often is only possible after one has gone through darkness.
允许路西法仍然保持一定量的自由度的原因，除了到目前为止经常解释的“他必须总能认同神圣法则的公正之处”之外，另一个原因是这是一个促进发展的必要手段。在很多情况下，邪恶（的本质）必须得到充盈的品尝，之后它才能通过自由意志和当事人自己的主动性而被克服。这种去克服的欲望必须通过每个个体灵魂中不断增加启迪来得到，而这通常只有在一个人经历了黑暗之后才成为可能。

22.47  That such enlightenment cannot come in one lifetime goes without saying. To accomplish the perfection that is needed to enter into the Kingdom of God—the perfection that was lost through the fall—and to shed all the darkness that has come upon a soul can never be done in one lifetime. Many, many lives or incarnations are indeed necessary. Life on earth is like a school where you develop from one class to the other. Sometimes you may stay for a while in one class and then you may have one or several incarnations in succession where you accomplish a great deal.
“这种类型的启迪无法在一次生命中就被得到”的事实似乎无需多言。要达到进入上帝的国度所需的完美——那种由于堕落而损失的完美——并且褪去所有那些灵魂上形成的黑暗，这永远是无法在一次生命中就达成的。许多，许多次生命或降世确实是有必要的。地球上的生活就像一个学校，在这里你从一个课堂发展到另一个课堂。有时候你可能会在同一个课堂中呆一段时间，然后你可能一次或连续数次降世，然后从中成就相当多的事。

22.48  Human beings who are incarnated from the world of darkness first come with very low and coarse instincts. Only after many incarnations and worked off karmic debts—and often after some suffering and a number of divine influences—will the attitude begin to change, slowly but surely. When the senses have begun to refine themselves a little bit, then the real work of self-finding and self-purification begins, and for this phase again, many incarnations, accompanied by changing conditions and circumstances, are necessary.
来自黑暗世界并进行降世的人们一开始通常带有非常低级、非常粗野的本能。只有在完成许多次降世并消除了因果业债之后——而且通常在一定量的痛苦经历和一定数量的神圣影响之后——他们的态度才会开始发生缓慢但坚定的转变。当他们的感觉开始对它们自身进行一点点的精炼时，这时候真正意义上的自我搜寻和自我净化才开始，而对于这个阶段来说，有必要进行许多次降世，这个过程还伴随着不断变化的条件和境况。

22.49  Even in this secondary phase, many beings have not yet the strength to find God in reality. Too much of the lower self is still present for them not to succumb to the influences of the Luciferic world, whether the influence comes in the form of direct inspiration or through unwitting human instruments. Then it will again take a considerable number of lives to awaken sufficiently to strengthen one’s willpower for the all-important process of self-purification. Only then will a further phase come in which the process of very gradual purification begins. In each life, conditions are prepared so that a certain side of the lower self has the opportunity to transform itself best.
哪怕在这个第二阶段中，许多存有们仍然还没有在现实中寻找上帝的力量。对他们来说，内心中仍然有太多的低级自我，以至于他们仍然会屈服于路西法派系世界的影响力，这种影响力要么以直接灵感启发的形式出现，要么通过无意中使用人类工具引发。这种情况下他们就要花费相当数量的生命轮回来再次充分觉醒，进而增强自己的意志力，最终开启至关重要的自我净化过程。只有这时才会进入下个阶段，使非常缓慢的净化过程真正开始。在每次生命中，所面对的境况都经过精心准备，从而创造出一种条件，使低级自我中的某个特定侧面有机会在最大限度上转化自己。

22.50  You see, it cannot be any other way, for it would be impossible to reach in one lifetime the necessary perfection needed to enter the Kingdom of God for always. With each life, even in the worst cases, something is gained, even if the benefits can only be fully experienced at a later period, when a being finally declares:  “My path leads to God. I will not listen to my lower self,” that self which is constantly and magnetically in touch with the world of darkness. The higher self, which is much further in the background and much more difficult to reach through all the layers of imperfections is, however, in constant contact with the divine world.
你看，事情不可能是任何另外样子的，因为要在一次生命中达到永远进入上帝国度的必要完美程度会是不可能完成的任务。在每次生命中，哪怕是最糟的情况下，也会获得某些收获，虽然说这些收获可能只能在之后的阶段中才能充分体验到，也就是当一个人最终宣称：“我的道路通往上帝。我不会听从我的低级自我，”的时候。而低级自我会持续地、像有磁性一样始终与黑暗世界保持接触。高级自我，它处于更远的背景中而且更加困难通过重重的不完美被联络到，是持续地与神圣世界保持接触的。

22.51  The outer personality, with its willpower and capability to decide one way or another, has the means one day to make the decisive step:  “I declare myself for God, for my higher self, with everything that it entails,” disregarding the laziness, the comfort, and the way of least resistance to give in to one’s faults. Whether the faults are still murder, stealing, wickedness or are now only selfishness, jealousy, envy, resentment, laziness, or whatever else, makes no difference in principle. Anyone truly declaring and deciding and remaining with the decision to follow the path to God—hence the salvation of Christ—cannot remain a subject of the Luciferic world. Lucifer will have no power over such a being whether on earth or in the spirit world.
人的外部人格，包括它的意志力和做决定的能力，有能力在某一天迈出决定性的一步：“我宣称自己为上帝，为我的高级自我服务，并接受其中一切内含的意义，”并且排除懒惰、安逸以及屈服于个人缺点的最小阻力原则。不管所要面对的缺点仍然是谋杀、偷窃、恶毒还是仅仅是自私、嫉妒、羡慕、怨恨、懒惰以及类似的任何类似问题，它们在原则上是没有区别的。任何真正宣告并决定并保持这个追随通往上帝的道路的决定的人——因此得到了基督的拯救——无法作为路西法派系世界的臣民保存在那里。路西法对这样一个存有将没有统治力，不管在地球上还是精神世界中都是如此。

22.52  This is the way Christ has opened the door. You may now understand why it is said that Christ saved you from your sins. This is accurate only in the sense that your great sin of falling, of not remaining faithful to God, and of becoming at one time part of the world of darkness does not have as a consequence eternal exclusion from the divine worlds. From this Christ has indeed saved you, and for this you certainly have all the reason in the world to be grateful to Him. Through Him you now have the possibility by your own efforts and development to cross the threshold. In that sense, it is correct to say that Christ died for your sins. However, the interpretation that Christ died for all your sins and all your faults is very wrong.
这就是基督打开门的方式。你们现在可以理解为什么说基督把你们从自己的罪中拯救了出来。这种说法仅仅从“你们发生堕落的重大罪、你们没有保持对上帝的忠实，以及在一段时间里成为了黑暗世界的一部分，不过后果并不是被永久排除在神圣世界之外”的意义上来说是准确的。从这一点来说基督确实拯救了你们，而为此你们当然有天底下所有的原因来对祂感到感激。通过祂，你们现在有机会通过自己的努力和发展来越过临界点。从这个意义上来说，“基督为了你们的罪而死”的说法是正确的。但是，那种说“基督为了你们所有的罪和所有的过失而死”的说法是非常错的。

22.53  This then very briefly is the story of the Creation of the Universe, the Fall, the Creation of this earth sphere, and salvation through Jesus Christ.
那么，这就是非常简略的关于宇宙造物，（天使）堕落，地球位面的创造，以及通过耶稣基督得到拯救的故事。

22.54  At the beginning of this series I encouraged you, my friends, to ask all the questions you have in mind about this subject that have not been answered in these lectures. I would like to suggest that you think about what I have told you. Reread my words, since you miss so much when you just hear a lecture for the first time. Then prepare your questions and I will gladly answer them, as well as any others.
在这系列讲座开始的时候我曾鼓励你们，我的朋友们，把你们脑中关于这个话题的所有那些没有在这些讲座中得到回答的问题都提出来。我想建议你们思考下我已经告诉你们的内容。重读我的话，因为当你们第一次听到一个讲座时会错过如此多的信息。然后准备好你们的问题，我会很高兴回答它们，以及任何其他问题。

22.55  Be blessed, my dear ones, be in God and in Christ.
愿你们受到祝福，我亲爱的人们，与上帝和基督同在。
_
_
©2019 International Pathwork Foundation
Translated by Mark.  Edited by April.
微信订阅号 @Pathwork Lectures 心路讲座中文
